src/vision/status_quo/alan_creates_a_hanging_alarm.md

```diff
@@ -0,0 +1,129 @@
+# 😱 Status quo stories: Alan Creates a Hanging Alarm
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today.
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+[Alan] is a developer on the Bottlerocket project.
+[Bottlerocket] is a Linux-based open-source operating system that is purpose-built by Amazon Web Services for running containers.
+We use our own program [pubsys] to ensure that Bottlerocket update repositories are healthy.
+
+### The TL;DR - What makes Alan Sad
+
+Although the first two of these points are about tokio, they are really about Rust async since tokio serves as the de-facto `std::runtime` for Rust.
+
+- It is confusing and dangerous that multiple tokio runtimes can panic or hang at program runtime.
+- It is challenging that using multiple major versions of tokio (which is allowed by cargo) can fail at runtime.
+- It is unfortunate that we need a 3rd party runtime in order to `block_on` a future, even if we are not trying to write async code.
+
+### Multiple Tokio Runtimes
+
+Bottlerocket uses its own [tough] library to read and write TUF repositories.
+This library was created before async became widespread and [reqwest] changed its main interface to `async`.
+When reqwest switched to async, Alan used the `reqwest::blocking` feature instead of re-writing tough to be an async interface.
+(Maybe Alan [should](https://github.com/awslabs/tough/issues/213) make tough an async interface, but he hasn't yet.)
+In order to provide a non-async interface, `reqwest::blocking` creates a tokio runtime so that it can await futures.
+
+In pubsys Alan created some parallel downloading logic while using the above libraries.
+Without realizing the danger, he created a tokio runtime in pubsys and used futures/await to do this parallelization.
+Surprisingly, in retrospect, this worked... until it didn't.
+
+Recently Alan discovered that his repository verification alarm was hanging.
+The root cause was having multiple tokio runtimes, though Alan don't know what change exposed the issue.
```

Comment:
Do you remember any details of how Alan figured this out?

Reply:
What's a "repository verification alarm" ?

Also, it's not totally clear to me if multiple tokios runtimes in pubsys related to the tokio runtime in bottlerocket